src/botPage/view/blockly/index.js

```diff
@@ -239,12 +237,16 @@ export const load = (blockStr, dropEvent = {}) => {
         const blockType = block.getAttribute('type');
 
         if (!Object.keys(Blockly.Blocks).includes(blockType)) {
-            const error = new TrackJSError(
-                'FileLoad',
-                translate('XML file contains unsupported elements. Please check or modify file.')
+            globalObserver.emit(
+                'Error',
+                createError(
+                    'InvalidBlockInXML',
+                    translate(
```

Comment:
your test link doesn't translate it, the problem is,  we have two `translate` functions one of them is in `i18n.js` file and the other is in the `common/tools.js` you should use `common/tools.js` function to translate contents with `variables`
currently, on your test link, it doesn't translate `{$0}` but if you use the correct translate function it returns the value of `{$0}`

Reply:
changed. imported another translate function. I did'n know about 2 function for translate :)